configs/pie-components-config/playwright/index.html

```diff
@@ -30,6 +30,7 @@
         }
         body {
             font-feature-settings: "tnum"; /* Enable tabular numbers */
+            font-family: JETSansDigital, monospace;
```

Comment:
Do you think we should potentially add this `font-family` definition into our wiki for what consumers should app to their `head` of applications? ([the integration bit](https://pie.design/foundations/typography/code/#implementation))